ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -1,10 +1,14 @@
 <div id="watched-repo-navbar" class="th-context-navbar watched-repo-navbar clearfix">
-  <th-watched-repo ng-repeat="(name, repoData) in repoModel.watchedRepos"></th-watched-repo>
+
+  <div style="display: flex; flex-direction: row-reverse; float: left;">
+    <th-watched-repo ng-repeat="(name, repoData) in repoModel.watchedRepos"></th-watched-repo>
```

Comment:
ng-repeat has an ``orderBy:reverse`` filter that lets you sort it.  That should work when you change ``watchedRepos`` to an array or set.